HW_5/HW_5_Task_3.js

```diff
@@ -0,0 +1,25 @@
+"use strict";
+// Создать программу, которая будет принимать на вход СЛОВО (создать переменную со словом), 
+//   и выводить в консоль количество гласных и согласных букв в этом слове. 
+//   Ответ должен выводиться шаблонным литералом вида word contains x vowels and y consonants
+
+const word = "СЛОВО".toUpperCase();
+const consonantsRU = "БВГДЖЗЙКЛМНПРСТФХЦЧШЩ"; // все согласные русского алфавита
+const vowelsRU = "АЕЁИОУЫЭЮЯ"; // все гласные русского алфавита
+const consonantsEN = "BCDFGHJKLMNPQRSTVWXYZ"; // все согласные английского алфавита
+const vowelsEN = "AEIOU"; // все гласные английского алфавита
+let numberOfConsonants = 0; // счетчик для согласных
+let numberOfvowels = 0; // счетчик для гласных
+for (let i = 0; i < word.length; i ++) {
+    if (consonantsRU.includes(word[i])) { // если содержит русскую согласную букву
+        numberOfConsonants += 1;
+    } else if (vowelsRU.includes(word[i])) { // если содержит русскую гласную букву
+        numberOfvowels += 1;
+    } else if (consonantsEN.includes(word[i])) { // если содержит английскую согласную букву
+        numberOfConsonants += 1;
+    } else if (vowelsEN.includes(word[i])) { // если содержит английскую гласную букву
+        numberOfvowels += 1;
+    }    
+}
```

Comment:
```suggestion
const word = "СЛОВО".toUpperCase();
const consonantsRU = "БВГДЖЗЙКЛМНПРСТФХЦЧШЩ"; // все согласные русского алфавита
const vowelsRU = "АЕЁИОУЫЭЮЯ"; // все гласные русского алфавита
const consonantsEN = "BCDFGHJKLMNPQRSTVWXYZ"; // все согласные английского алфавита
const vowelsEN = "AEIOU"; // все гласные английского алфавита
let numberOfConsonants = 0; // счетчик для согласных
let numberOfvowels = 0; // счетчик для гласных
for (let i = 0; i < word.length; i ++) {
    if (consonantsRU.includes(word[i]) || consonantsEN.includes(word[i])) { 
        numberOfConsonants += 1;
    } else if (vowelsRU.includes(word[i]) || vowelsEN.includes(word[i])) { 
        numberOfvowels += 1;
    } 
}
```

Можно чуть сократить код:)